templates/cluster-template-ephemeral.yaml

```diff
@@ -169,11 +167,9 @@ spec:
     spec:
       location: ${AZURE_LOCATION}
       osDisk:
-        diffDiskSettings:
-          option: Local
         diskSizeGB: 128
         managedDisk:
-          storageAccountType: Standard_LRS
+          storageAccountType: Premium_LRS
```

Comment:
This seems unrelated updating to 1.16. Was this needed or is related to another change set?

Reply:
I think it happened somehow from the generate, i'll look into it

Reply:
I might have messed up a rebase 🤷